frontend/src/app/app.module.ts

```diff
@@ -45,12 +45,16 @@ import { OrganizationOffersListComponent } from './organization/organization-off
 
 Raven.config(environment.sentryDSN).install();
 
-export class RavenErrorHandler implements ErrorHandler {
+class RavenErrorHandler implements ErrorHandler {
   handleError(err: any): void {
     Raven.captureException(err);
   }
 }
 
+export function ErrorHandlerFactory(): ErrorHandler | RavenErrorHandler {
```

Comment:
Do we need to have `ErrorHandler | RavenErrorHandler` here? `RavenErrorHandler` implements `ErrorHandler`

Reply:
You are right, I changed it to ErrorHandler type only.